app/components/place-holder.js

```diff
@@ -137,8 +143,27 @@ export default Ember.Component.extend({
         },
 
         changeChart: function(chart){
+
```

Comment:
this is a workaround until the aggregation parsing is done, let's put a comment in to let people know